internal/service/ocm/ocm_interface.go

```diff
@@ -0,0 +1,32 @@
+package ocm
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/redhatinsights/mbop/internal/models"
+)
+
+type OCM interface {
+	InitSdkConnection(ctx context.Context) error
+	CloseSdkConnection()
+	GetUsers(users models.UserBody, q models.UserQuery) (models.Users, error)
+	GetOrgAdmin([]models.User) (models.OrgAdminResponse, error)
+}
+
+func NewOcmClient() (OCM, error) {
+	var client OCM
+
+	mod := os.Getenv("OCM_MODULE")
```

Comment:
We should be able to use @lindgrenj6's new config module changes here: https://github.com/RedHatInsights/mbop/pull/33/

Reply:
Disregard! :) Looking at individual commits here bit me.

Reply:
And it will also need the new `ams` module constant check as well, I think? :)

Reply:
And will also need to use `USERS_MODULE`?